docs/1-the-manual-menace/1-the-basics.md

```diff
@@ -41,10 +44,11 @@ echo ${CLUSTER_DOMAIN}
 echo ${GIT_SERVER}
 ```
 
-8. Check if you can connect to OpenShift. Run the command below. 
+8. Check if you can connect to OpenShift. Run the command below which should result in a "Login successful" reply. Don't forget to make your replacements for \<USERNAME> and \<PASSWORD>. 
```

Comment:
I raised the same issue - and a PR for this.  
This wording is probably better though!